docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -11,292 +11,27 @@ When viewing an asset in Dagster’s UI, you can see all of its checks, and whet
-    Definitions,
-    asset,
-    asset_check,
-)
-
-
-@asset
-def orders(): ...
-
-
-@asset
-def items(): ...
-
-
-def make_check(check_blob: Mapping[str, str]) -> AssetChecksDefinition:
-    @asset_check(
-        name=check_blob["name"],
-        asset=check_blob["asset"],
-        required_resource_keys={"db_connection"},
-    )
-    def _check(context):
-        rows = context.resources.db_connection.execute(check_blob["sql"])
-        return AssetCheckResult(passed=len(rows) == 0, metadata={"num_rows": len(rows)})
-
-    return _check
-
-
-check_blobs = [
-    {
-        "name": "orders_id_has_no_nulls",
-        "asset": "orders",
-        "sql": "select * from orders where order_id is null",
-    },
-    {
-        "name": "items_id_has_no_nulls",
-        "asset": "items",
-        "sql": "select * from items where item_id is null",
-    },
-]
-
-defs = Definitions(
-    assets=[orders, items],
-    asset_checks=[make_check(check_blob) for check_blob in check_blobs],
-    resources={"db_connection": MagicMock()},
-)
-```
+TODO
 
 ---
 
-## Executing checks
-
-### Via the UI
-
-Materializing an asset from the UI will also execute any checks that are defined for that asset. You can also execute checks without materializing the asset from the Checks tab of the asset’s detail page.
-
-### Via sensors and schedules
-
-You can use <PyObject object="define_asset_job" /> to define jobs that execute sets of both assets and checks, and then trigger those jobs via [sensors](/concepts/partitions-schedules-sensors/sensors) or [schedules](/concepts/partitions-schedules-sensors/schedules). By default, checks are included with the assets they check. You can also define jobs that include only checks, or only assets.
-
-```python file=/concepts/assets/asset_checks/jobs.py
-from dagster import (
-    AssetSelection,
-    Definitions,
-    ScheduleDefinition,
-    asset,
-    asset_check,
-    define_asset_job,
-)
-
-
-@asset
-def my_asset(): ...
-
-
-@asset_check(asset=my_asset)
-def check_1(): ...
-
-
-@asset_check(asset=my_asset)
-def check_2(): ...
-
-
-# includes my_asset and both checks
-my_job = define_asset_job("my_job", selection=AssetSelection.assets(my_asset))
-
-
-# includes only my_asset
-my_asset_only_job = define_asset_job(
-    "my_asset_only_job",
-    selection=AssetSelection.assets(my_asset).without_checks(),
-)
-
-# includes check_1 and check_2, but not my_asset
-checks_only_job = define_asset_job(
-    "checks_only_job", selection=AssetSelection.checks_for_assets(my_asset)
-)
-
-# includes only check_1
-check_1_job = define_asset_job("check_1_job", selection=AssetSelection.checks(check_1))
-
-# schedule my_job to run every day at midnight
-basic_schedule = ScheduleDefinition(job=my_job, cron_schedule="0 0 * * *")
-
-defs = Definitions(
-    assets=[my_asset],
-    asset_checks=[check_1, check_2],
-    jobs=[my_job, my_asset_only_job, checks_only_job, check_1_job],
-    schedules=[basic_schedule],
-)
-```
-
----
+## Uses
```

Comment:
Maybe pedantic, but I'm struggling a little bit with the word "Uses" here.

I think there are a few high-level facts that are important to communicate about how asset checks work:
- They're defined in code
- There are lots of ways to define them - Python, dbt, etc.
- They can be executed in the same way assets and jobs can, i.e. by using the Dagster UI or with schedules/sensors
- They can optionally be used to block downstream execution
- They have severity levels and metadata that help communicate actionable details about the results
- You can create alerts with them in Dagster Cloud

but I'm finding it difficult to frame many of these as a "use".

"Uses" and "Benefits" also feel overlapping. E.g. "Uses of a rake: clean my lawn", "Benefits of a rake: my lawn is clean". Technically different, but communicate the same information.

Which is all to say, I wonder if a header like "Summary" or "How they work" or "How they're used" would make sense?

Reply:
I took another swing at this - lmk what you think